source/classes.tex

```diff
@@ -1445,17 +1445,20 @@
 \end{note}
 
 \pnum
-\indextext{constructor!implicitly called}%
-Default constructors are called implicitly to create class objects of static, thread,
-or automatic storage duration~(\ref{basic.stc.static}, \ref{basic.stc.thread}, \ref{basic.stc.auto}) defined
-without an initializer\iref{dcl.init},
-are called to create class objects of dynamic storage duration\iref{basic.stc.dynamic} created by a
-\grammarterm{new-expression}
-in which the
-\grammarterm{new-initializer}
-is omitted\iref{expr.new}, or
-are called when the explicit type conversion syntax\iref{expr.type.conv} is
-used.
+\indextext{constructor!implicitly invoked}%
+A default constructor is invoked implicitly
+\begin{itemize}
+\item
+to initialize a class object of static, thread,
+or automatic storage duration~(\ref{basic.stc.static}, \ref{basic.stc.thread}, \ref{basic.stc.auto})
+defined without an initializer\iref{dcl.init},
+\item
+to initialize a class object of dynamic storage duration\iref{basic.stc.dynamic}
+created by a \grammarterm{new-expression}
+in which the \grammarterm{new-initializer} is omitted\iref{expr.new}, or
+\item
+when the explicit type conversion syntax\iref{expr.type.conv} is used.
+\end{itemize}
 A program is ill-formed if the default constructor for an object
 is implicitly used and the constructor is not accessible\iref{class.access}.
```

Comment:
Maybe replace "used" with "invoked" to make it clear that this is talking about the same set of things we just mentioned.

Please consider reordering the note after the normative sentence. As it stands, it sounds like the normative rule uses a term defined in the note; reordering the two would give the (presumably intended) impression that the note is just a summary of things said elsewhere.

Reply:
used -> invoke done

I don't understand the second part of the comment; the notes in the vicinity are totally unrelated.